app/app.go

```diff
@@ -647,10 +652,10 @@ func NewWasmApp(
 		wasmOpts...,
 	)
 
-	// Create Transfer Stack
-	var transferStack porttypes.IBCModule
-	transferStack = transfer.NewIBCModule(app.TransferKeeper)
-	transferStack = ibcfee.NewIBCMiddleware(transferStack, app.IBCFeeKeeper)
+	// Create fee enabled wasm ibc Stack
+	var wasmStack porttypes.IBCModule
+	wasmStackIBCHandler := wasm.NewIBCHandler(app.WasmKeeper, app.IBCKeeper.ChannelKeeper, app.IBCFeeKeeper)
+	wasmStack = ibcfee.NewIBCMiddleware(wasmStackIBCHandler, app.IBCFeeKeeper)
```

Comment:
MaxIBCCallbackGas is not supposed to be used for this stack?

Reply:
No, this is our module, wrapped in the ibcfee middleware (not callbacks). The diff just looks a bit confusing on this.

We could implement support for the callbacks middleware at some point in the future though. But that requires some more thought on how it should work.

Reply:
Not sure I understand. The fee middleware is already in main for a long time. I only moved the wasm stack up because I need a reference to the `wasmStackIBCHandler` for the other stacks.

Reply:
Sorry, then everything is all good.